website/docs/d/atracker_endpoints.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_atracker_endpoints"
+description: |-
+  Get information about atracker_endpoints
+subcategory: "Activity Tracking API"
```

Comment:
Can we change the sub category : Atracker or to the service name

Reply:
It has been decided that ATracker will not be  used as a display name. I will update the ATracker.yaml file to change all references of `Activity Tracking` to `Activity Tracker` and regen the code.

Reply:
Regenerated code should now have `Activity Tracker`